### PR TITLE
Fix samples test projects

### LINKED ISSUE #9707: Opt RemoveByValVB into the default test app.config so it avoids the EventListener bug
http://dotnet-ci.cloudapp.net/job/roslyn_future-stabilization_win_rel_unit32/22/artifact/Binaries/Release/xUnitResults/RemoveByValVB.UnitTests.dll.out

### PR DESCRIPTION
**Pull request by Pilchie**
Remove unneeded app.config files from samples test projects.
Add missing xunit toolset imports to samples test projects.

Fixes #9707.

### CODE REVIEW

**Comment by Pilchie** [comment]
Not sure why @DustinCampbell's commits are showing up in this PR, but whatever.

Tagging @dotnet/roslyn-infrastructure and @dotnet/roslyn-ide

File: src/Samples/VisualBasic/RemoveByVal/Test/app.config [hunk 1]
```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-  <appSettings>
-    <add key="xunit.useAppDomain" value="true"/>
```
Comment (Pilchie) [inline src/Samples/VisualBasic/RemoveByVal/Test/app.config]: This line here was the bad one.

**Comment by davkean** [comment]
:+1: 

You likely based it off stabilization, but are merging into master.

**Comment by Pilchie** [comment]
I ran:
```
git fetch --all
git merge-base upstream/stabilization upstream/master
git checkout -b SamplesAppConfig <sha from line above>
```
And did the work from there.  I would have expected that to give me a parent commit that would allow me to cleanly merge into either `stabilization` or `master`.

**Comment by davkean** [comment]
Did we accidently throw away these changes in a later merge?

**Comment by Pilchie** [comment]
Summoning @jasonmalinowski and @jaredpar for their git-fu.

Maybe throw @shiftkey in too just for fun.

**Comment by davkean** [comment]
Oh, no, maybe GitHub is showing based on the master head *before* the merge.

**Comment by Pilchie** [comment]
Note also that the *content* of d1ec2d9c691f9dc49dd1d12a6ebc055b353e1522 doesn't seem to be in my diff, even though GH says it's being merged.  Is that because the content was already there as the result of a cherry-pick?

**Comment by shiftkey** [comment]
Footnote: using [pretty log](https://coderwall.com/p/euwpig/a-better-git-log) I can see @Pilchie's branch started at an earlier commit than `master` 6049f4f (with Dustin's commits included):



If I reset my local `master` back to the commit before the PR, i.e. e42ba20f77c6e23c55d23917ac9ee886f3910cf2, and then run `git merge-base` it gives me 44b031971ac9f5b6e75881d228f50bb5ff0f46d1, which is far removed from `master`:

```
> git merge-base e42ba20f origin/stabilization 
44b031971ac9f5b6e75881d228f50bb5ff0f46d1
> git log e42ba20...44b03197 --oneline | wc -l
     359
> git log e42ba20...6049f4f --oneline | wc -l
     363
> git log 44b03197...6049f4f --oneline | wc -l 
       4
```

However it's rather close to `6049f4f`, so I guess that's how it was chosen. It's a commit from a few days ago, but given the proximity it seems a logical choice for for `master` and `stabilization` when the branch was created.

The thing I can't quite explain is where Dustin's commits were introduced, as those only seem to have existed in the `SamplesAppConfig` branch...

**Pull request merged**